applications/tari_console_wallet/src/automation/commands.rs

```diff
@@ -652,6 +685,44 @@ pub async fn command_runner(
                 },
                 Err(e) => eprintln!("CreateKeyPair error! {}", e),
             },
+            CreateAggregateSignatureUtxo(args) => match create_aggregate_signature_utxo(
+                transaction_service.clone(),
+                args.amount,
+                args.fee_per_gram,
+                args.n,
+                args.m,
+                args.public_keys
+                    .iter()
+                    .map(|pk| PublicKey::from(pk.clone()))
+                    .collect::<Vec<_>>(),
+                args.message,
+            )
+            .await
+            {
+                Ok((tx_id, output_hash)) => {
+                    println!(
+                        "Create a utxo with n-of-m aggregate public key, with: 
+                            1. n = {},
+                            2. m = {}, 
+                            3. tx id = {},
+                            4. output hash = {}",
+                        args.n, args.m, tx_id, output_hash
+                    )
+                },
+                Err(e) => eprintln!("CreateAggregateSignatureUtxo error! {}", e),
+            },
+            SignMessage(args) => match sign_message(args.private_key, args.challenge) {
+                Ok((sgn, nonce)) => {
+                    println!(
+                        "Sign message: 
+                                1. signature: {},
+                                2. public key: {}",
+                        *Zeroizing::new(sgn.get_signature().to_hex()),
```

Comment:
We dont have to Zeroize this, the signature is public anyway. 
The argument private key is not thou.

Reply:
Resolved !